### PR TITLE
Implemented The login/home button going off on login.

### PR DESCRIPTION
**Pull request by adetayo2718**
Screenshot 2024-04-15 224442
Screenshot 2024-04-15 224539

Implemented The login/home button going off on login.

### CODE REVIEW

File: src/Layout/components/Header.tsx [hunk 1]
```diff
@@ -35,11 +39,11 @@ const Header = () => {
               <span onClick={handleLogout} className="cursor-pointer">
                 Logout
               </span>
-              <NavLink to="/signin">Log in</NavLink>
+              {userLoggedIn ? "" : <NavLink to="/signin">Log in</NavLink>}
```
Comment (pleasantvik) [inline src/Layout/components/Header.tsx]: Why the ternary comparison here. Use the && shortciruit

File: src/Layout/components/Header.tsx [hunk 2]
```diff
@@ -35,11 +39,11 @@ const Header = () => {
...
             </>
           </div>
           <div className="bottom flex gap-4 justify-end">
-            <NavLink to="/">Home</NavLink>
+            {userLoggedIn ? "" : <NavLink to="/">Home</NavLink>}
```
Comment (pleasantvik) [inline src/Layout/components/Header.tsx]: Use the && shortciruit operation

`    {!userLoggedIn && <NavLink to="/">Home</NavLink>}
`

The logout button should not show when someone is not logged in also

**Review by pleasantvik — COMMENTED** [review]
MAke the neccessary changes

**Review by pleasantvik — APPROVED** [review]
Great work

File: src/Layout/components/Header.tsx (hunk 1 above)
Comment (adetayo2718) [reply src/Layout/components/Header.tsx]: I have just resolved it

File: src/Layout/components/Header.tsx (hunk 2 above)
Comment (adetayo2718) [reply src/Layout/components/Header.tsx]: Its been resolved

**Pull request merged**